README.md

```diff
@@ -90,3 +90,91 @@ $ docker build . -t <IMAGE_TAG>
 ### Format
 
 Formatting for `PrivateKey` or `Peer` follows the format in [Nekoyume Project README][../README.md].
+
+## How to run NineChronicles Standalone on AWS EC2 instance using Docker
+
+### A) On Your Local
+
+#### Pre-requisites
+
+- Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
+- Docker hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
+
+#### 1. Build docker image with the tag name in [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] format.
+
+```
+$ docker build . --tag 9c/9c-standalone --build-arg COMMIT=9c-1
+
+Usage: docker build . --tag [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] --build-arg COMMIT=[<VERSION_SUFFIX>]
+```
+- [Docker Build Guide](https://docs.docker.com/engine/reference/commandline/build/)
+
+![Docker Build](https://i.imgur.com/vc6CnbR.png)
+
+#### 2. Push your docker image to your docker hub account.
+
+```
+$ docker push 9c/9c-standalone:latest
+
+Usage: docker push [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] : [<VERSION>]
+```
+- [Docker Push Guide](https://docs.docker.com/engine/reference/commandline/push/)
+
+![Docker Push](https://i.imgur.com/IRIXXjg.png)
+
+### B) On Your AWS EC2 Instance
+
+#### Pre-requisites
+
+- Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
+- Docker hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
+- AWS EC2 instance: [AWS EC2 Guide](https://docs.aws.amazon.com/ec2/index.html)
+
+#### 3. Pull your docker image to your AWS EC2 instance.
+
+```
+$ docker pull 9c/9c-standalone:latest
+
+Usage: docker pull [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] : [<VERSION>]
+```
+- [Docker Pull Guide](https://docs.docker.com/engine/reference/commandline/pull/)
+
+![Docker Pull](https://i.imgur.com/e7sSKxH.png)
+
+#### 4. Create a docker volume for blockchain data persistance
+
+```
+$ docker volume create 9c-volume
+Usage: docker volume create [<VOLUME_NAME>]
+```
+- [Docker Volume Guide](https://docs.docker.com/engine/reference/commandline/volume_create/)
+
+![Docker Volume Create](https://i.imgur.com/ISgKeLc.png)
+
+#### 5. Run your docker image with your docker volume mounted (use -d for detached mode)
+
+```
+$ docker run \
+--detach \
+--volume 9c-volume:/app/data \
+9c/9c-standalone \
```

Comment:
Following commands can be shown in `README.md` and `dotnet run --help`... idk it is good idea or not to write all arguments here.

Reply:
Ok I'll replace the long standalone options with [9c-standalone options] and then direct it to the option descriptions on top for reference!